Metro-Records/la-metro-councilmatic
# Refactor index and about, upgrade bootstrap

## Overview

This upgrades bootstrap to version 5 and refactors the index and about pages

Connects #969

### Notes

I have this comparing to the `bootstrap-mobile-adjustments` branch so that they can be merged into that piece by piece, and then this bootstrap branch can be merged into main once everything is done.

If there's a better way to do this, please lmk!

## Testing Instructions

 * Check the index and about pages against their live versions on desktop and mobile window sizes
 * Confirm that
   * they look presentable/same
   * the nav works
   * the functionality across those two pages still works (tooltips, table of contents, etc)
